egg.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "ramda",
+  "description": "A practical functional library for JavaScript programmers.",
+  "stable": true,
+  "entry": "./source/index.js",
+  "files": [
+    "./source/**/*.js",
+    "./README.md"
+  ],
+  "version": "0.27.0",
```

Comment:
Will this need to be manually updated when we do releases?

Reply:
That would be ideal; but my understanding is that new versions needs to be published any. I can take care of that.

Reply:
I think we can include that in our build script.

Reply:
@CrossEye It would be very easy for you all to connect this to our `publish` workflow [here](https://github.com/nestdotland/eggs-ci) for new releases!

Reply:
@tbaumer22: Thanks.  We'll take a look.